src/adapters/FallbackAdapter.php

```diff
@@ -22,6 +22,10 @@ protected function init()
 
     public function getCode()
     {
-        return Utils::iframe($this->url);
+        if (!$this->url) {
```

Comment:
It might be late and I just got off work but this is invert for and empty URL cos it's the fallback correct?

Reply:
yes, if its an empty url then it doesn't return anything and therefore won't show the site homepage in the preview